cf/data/array/fullarray.py

```diff
@@ -0,0 +1,185 @@
+import numpy as np
+
+from .abstract import Array
+
+
+class FullArray(Array):
+    """A array filled with a given value.
+
+    The array may be empty or all missing values.
+
+    """
+
+    def __init__(
+        self,
+        fill_value=None,
+        dtype=None,
+        shape=None,
+        units=False,
+        calendar=False,
+        source=None,
+        copy=True,
+    ):
+        """**Initialization**
+
+        TODODASKDOCS
+
+            :Parameters:
+
+                dtype : numpy.dtype
+                    The numpy data type of the data array.
+
+            shape : tuple
+                The data array's dimension sizes.
```

Comment:
I see the `TODODASKDOCS` above but might as well fix the parameters already listed...

```suggestion
                shape : tuple
                    The data array's dimension sizes.
```